bin/deploy.sh

```diff
@@ -81,78 +71,21 @@ if [[ -z $BRANCH ]]; then
     fi
 fi
 
-old_head=$(git rev-parse origin/$BRANCH)
-
 update_repo
-new_head=$(git rev-parse origin/$BRANCH)
+activate_once
 
+echo "Updating python dependancies"
+env --unset GIT_WORK_TREE pip install --quiet --requirement requirements.txt
 
-# New modules
-if [[
-    $FORCE ||
-    ( -n $(git diff $old_head $new_head -- ${GIT_WORK_TREE}/setup.py) && $? -eq 0 ) ||
-    ( -n $(git diff $old_head $new_head -- ${GIT_WORK_TREE}/requirements.txt) && $? -eq 0 )
-]]; then
-    activate_once
-
-    if [[ $VERBOSE ]]; then
-        echo "Updating python dependancies"
-    fi
-    cd "${GIT_WORK_TREE}"
-    env -u GIT_WORK_TREE pip install --requirement requirements.txt
+echo "Synchronizing database"
+flask sync
 
-    # Restart in case celery module updates
-    sudo service celeryd restart
-fi
-
-# DB Changes
-if [[
-    ($FORCE || ( -n $(git diff $old_head $new_head -- ${GIT_WORK_TREE}/portal/migrations) && $? -eq 0 ))
-]]; then
-    activate_once
-
-    if [[ $VERBOSE ]]; then
-        echo "Running database migrations"
-    fi
-
-    # Prevent config bootstrap issue by removing old config
-    rm --verbose ${GIT_WORK_TREE}/instance/site.cfg
-    flask sync
-fi
-
-# New seed data
-if [[ $FORCE || $SEED || ( -n $(git diff $old_head $new_head -- ${GIT_WORK_TREE}/portal/models) && $? -eq 0 ) ]]; then
-    activate_once
-    echo "Seeding database"
-    flask seed
-fi
-
-
-# Celery Changes
-if [[ $FORCE || ( -n $(git diff $old_head $new_head -- ${GIT_WORK_TREE}/portal/tasks.py) && $? -eq 0 ) ]]; then
-    activate_once
-
-    if [[ $VERBOSE ]]; then
-        echo "Restarting celeryd"
-    fi
-    sudo service celeryd restart
-fi
-
-# Code changes - update package metadata
-if [[ $FORCE || ( -n $(git diff $old_head $new_head -- ${GIT_WORK_TREE}) && $? -eq 0 ) ]]; then
-    activate_once
-
-    if [[ $VERBOSE ]]; then
-        echo "Updating package metadata"
-    fi
-    cd "${GIT_WORK_TREE}"
-    python setup.py egg_info
-fi
+echo "Updating package metadata"
+python setup.py egg_info --quiet
 
 # Restart apache if application is served by apache
 if [[ "${GIT_WORK_TREE}" == "/srv/www/"* ]]; then
-    if [[ $VERBOSE ]]; then
-        echo "Restarting apache"
-    fi
+    echo "Restarting services"
     sudo service apache2 restart
+    sudo service celeryd restart
```

Comment:
We should be able to get that from the current path pretty easily. I'll circle back once Adam finalizes the celerybeat work